backend/src/etc/clean-words.py

```diff
@@ -0,0 +1,25 @@
+# Adam Fernandes
+# ShellHacks 2020
+# Quick and dirty script for cleaning previous data
+
+# Read file
+fileObject = open("keywords.txt", "r")
+
+rawInput = fileObject.readlines()
```

Comment:
Since youre opening the file as a list and not a string, add this function to FileHelpers.py:
```
def load_file_as_list(file_path: str) -> str:
    """
    Loads the file at a specificed path, returns an error otherwise
        
    :param file_path (str): The path to the file you want to load
    :return _file a str of the contents of the file
    """
    _file = ""
    try:
        with open(file_path, "r") as fp:
            _file = fp.readlines()
        return _file
    except Exception:
        raise Exception(f"Error reading file at: {file_path}")
```